modules/ossm-configuring-external-jaeger.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/customizing-installation-ossm.adoc
+
+
+[id="ossm-specifying-external-jaeger_{context}"]
+= Specifying an external Jaeger
+
+You can configure and deploy a standalone Jaeger instance and then specify the `name` of the Jaeger resource as the value for `spec.addons.jaeger.name` in the `ServiceMeshControlPlane` resource.  If a Jaeger resource matching the value of `name` exists, the control plane will use the existing installation.  This approach lets you fully customize your Jaeger configuration.
```

Comment:
here, Do we have to mention to read the best practices?
As the jaeger should be installed on the same namespace.

Reply:
Not sure we have a defined recommended practice that can be applied to all cases. Basically:

- if tenants are separated by namespaces, it would be good practice to have one Jaeger per namespace
- for small to mid-size, single-tenant clusters, one Jaeger in the `observability` namespace would be recommended